enhancements/sdk-migrate-bundle.md

```diff
@@ -0,0 +1,327 @@
+---
+title: sdk-migrate-bundle
+authors:
+  - "@jmrodri"
+reviewers:
+  - "@estroz"
+  - "@ecordell"
+  - "@rashmigottipati"
+approvers:
+  - "@estroz"
+  - "@rashmigottipati"
+creation-date: 2021-02-08
+last-updated: 2021-02-08
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Migrate Bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions [optional]
+
+1. The bundles we build today are done via `docker` from the project's
+   `Makefile` where users can adapt their image builder to whatever they want.
+   Do we want to allow this from the CLI?
```

Comment:
If we choose to build images and use opm to do that, it may be a good idea to allow this from the CLI as opm takes in the imageBuilder argument (ex: `docker` or `podman`) when building bundle manifests image. This gives users flexibility to pick their own image builder.